tests/dracula/test_run_topp_jnt_crt.py

```diff
@@ -11,36 +12,49 @@
 class TestRunToppJntCrt(unittest.TestCase):
     """Class for unit test."""
 
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
-    waypts_jnt = np.loadtxt("test_waypts_jnt_0.txt")
-    lim_ones = np.ones(waypts_jnt.shape[1])
+    paths = sorted(
+        glob.glob(
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                "test_waypts_crt_*.txt",
+            )
+        )
+    )
+    waypts_jnt_list = [
+        np.loadtxt(path.replace("_crt_", "_jnt_")) for path in paths
+    ]
+    waypts_crt_list = [np.loadtxt(path) for path in paths]
+    lim_ones = np.ones(waypts_jnt_list[0].shape[1])
     lim_ones = np.vstack([-lim_ones, lim_ones]).T
     vlim_jnt = 1 * lim_ones  # rad/s
     alim_jnt = 7.5 * lim_ones  # rad/s^2
-    waypts_crt = np.loadtxt("test_waypts_crt_0.txt")
-    lim_ones = np.ones(waypts_crt.shape[1])
+    lim_ones = np.ones(waypts_crt_list[0].shape[1])
     lim_ones = np.vstack([-lim_ones, lim_ones]).T
     vlim_crt = 1 * lim_ones  # m/s
     alim_crt = 10 * lim_ones  # m/s^2
 
     def test_run_topp_jnt_crt(self):
         """Test run_topp_jnt_crt(), just assert no throw."""
```

Comment:
Can you add some more detail on this test case. What is being tested at a high level, where does the test data come from, what is the expected output?

Reply:
yes, added more description to docstring in 
a493a07